contrib/craftscripts/maze.js

```diff
@@ -21,363 +21,369 @@ importPackage(Packages.com.sk89q.worldedit);
 importPackage(Packages.com.sk89q.worldedit.blocks);
 importPackage(Packages.com.sk89q.worldedit.math);
 
-usage = "<block> [width] [length] [height] [size] [thickness] flags\n";
-usage += "\n";
-usage += "Maze Flags:\n";
-usage += "• i adds an entry and an exit\n";
-usage += "• y places the entry and the exit randomly\n";
-usage += "• f adds a floor\n";
-usage += "• c adds a ceiling\n";
-usage += "• e places air blocks\n";
-usage += "• a places air blocks only\n";
-usage += "• v creates a vertical maze\n";
-usage += "\n";
-usage += "Solver Flags:\n";
-usage += "• s enables the maze solver\n";
-usage += "• g places glass if wrong or unvisited\n";
-usage += "• r places red wool if wrong\n";
-usage += "• b places blue wool if unvisited";
-
-context.checkArgs(1, -1, usage);
+usage = "Usage :\n";
+usage += "§6/.s <block> [width] [length] [height] [size] [thickness] flags\n";
```

Comment:
If we're going to add color I would rather not do it with the legacy color codes. Craftscripts should be able to use the text libraries just fine.